examples/vumps/vumps_hubbard_extended.jl

```diff
@@ -1,5 +1,6 @@
 using ITensors
 using ITensorInfiniteMPS
+using Revise
```

Comment:
We should remove this, you can load it by default on your own system (https://timholy.github.io/Revise.jl/stable/config/#Using-Revise-by-default-1).